ansible/roles/nomis-weblogic/tasks/install-healthcheck.yml

```diff
@@ -32,4 +44,4 @@
       changed_when: false
 
   # block
-  when: not weblogic_healthcheck_installed_check.stat.exists
+  # when: not weblogic_healthcheck_installed_check.stat.exists
```

Comment:
Enable healthcheck needs to be enabled as well

Reply:
Oops, indeed, forgot to revisit this file when I commented this step out in the debug process